src/ASM/DomainDecomposition.C

```diff
@@ -876,9 +876,9 @@ bool DomainDecomposition::calcGlobalEqNumbers(const ProcessAdm& adm,
 
     std::vector<int> LM;
     size_t nMult = 0;
-    for (size_t n = 0; n < locLMs.size(); ++n) {
+    for (size_t nn = 0; nn < locLMs.size(); ++nn) {
```

Comment:
Could use something like `for (int node : locLMS)) {`here

Reply:
sure, changed to ranged for.